tests/js/spec/stores/guideStore.spec.jsx

```diff
@@ -48,11 +49,33 @@ describe('GuideStore', function() {
     GuideStore.onFetchSucceeded(data);
     // GuideStore should prune steps that don't have anchors.
     expect(GuideStore.state.currentGuide.steps).toHaveLength(2);
+    expect(GuideStore.state.currentGuide.seen).toEqual(false);
     GuideStore.onNextStep();
     expect(GuideStore.state.currentStep).toEqual(1);
     GuideStore.onNextStep();
     expect(GuideStore.state.currentStep).toEqual(2);
     GuideStore.onCloseGuideOrSupport();
-    expect(GuideStore.state.guidesSeen).toEqual(new Set([1]));
```

Comment:
Instead of removing this make sure guide.seen is set to false, otherwise you're not really testing the close

Reply:
fixed